backend/src/main/java/gov/cdc/usds/simplereport/validators/TestResultFileValidator.java

```diff
@@ -21,202 +21,216 @@
+        errors.addAll(icu.getPossibleError());
+        errors.addAll(orderingFacilityName.getPossibleError());
+        errors.addAll(orderingFacilityStreet.getPossibleError());
+        errors.addAll(orderingFacilityStreet2.getPossibleError());
+        errors.addAll(orderingFacilityCity.getPossibleError());
+        errors.addAll(orderingFacilityState.getPossibleError());
+        errors.addAll(orderingFacilityZipCode.getPossibleError());
+        errors.addAll(orderingFacilityPhoneNumber.getPossibleError());
+        errors.addAll(comment.getPossibleError());
+        errors.addAll(testResultStatus.getPossibleError());
+
+        // validate individual values
+        errors.addAll(validateState(patientState));
+        errors.addAll(validateState(orderingProviderState));
+        errors.addAll(validateState(testingLabState));
+        errors.addAll(validateState(orderingFacilityState));
+
+        errors.addAll(validateZipCode(patientZipCode));
+        errors.addAll(validateZipCode(orderingProviderZipCode));
+        errors.addAll(validateZipCode(testingLabZipCode));
+        errors.addAll(validateZipCode(orderingFacilityZipCode));
+
+        errors.addAll(validatePhoneNumber(patientPhoneNumber));
+        errors.addAll(validatePhoneNumber(orderingProviderPhoneNumber));
+        errors.addAll(validatePhoneNumber(testingLabPhoneNumber));
+        errors.addAll(validatePhoneNumber(orderingFacilityPhoneNumber));
+
+        errors.addAll(validateDate(patientDob));
+        errors.addAll(validateDate(illnessOnsetDate));
+
+        errors.addAll(validateDateTime(orderTestDate));
+        errors.addAll(validateDateTime(specimenCollectionDate));
+        errors.addAll(validateDateTime(testingLabSpecimenReceivedDate));
+        errors.addAll(validateDateTime(testResultDate));
+        errors.addAll(validateDateTime(dateResultReleased));
+
+        errors.addAll(validateEmail(patientEmail));
+        errors.addAll(validateRace(patientRace));
+        errors.addAll(validateBiologicalSex(patientGender));
+        errors.addAll(validateEthnicity(patientEthnicity));
+
+        errors.addAll(validateYesNoAnswer(pregnant));
+        errors.addAll(validateYesNoAnswer(employedInHealthcare));
+        errors.addAll(validateYesNoAnswer(symptomaticForDisease));
+        errors.addAll(validateYesNoAnswer(residentCongregateSetting));
+        errors.addAll(validateYesNoAnswer(hospitalized));
+        errors.addAll(validateYesNoAnswer(icu));
+        errors.addAll(validateResidence(residenceType));
+
+        errors.addAll(validateTestResult(testResult));
+        errors.addAll(validateTestResultStatus(testResultStatus));
+        errors.addAll(validateSpecimenType(specimenType));
+
+        errors.addAll(validateClia(testingLabClia));
+      }
+    } catch (CsvProcessingException ex) {
+      log.error("Unable to parse test result csv.", ex);
+      errors.add(
+          new FeedbackMessage(
+              "File",
```

Comment:
I think we want to use this scope here:
https://github.com/CDCgov/prime-simplereport/blob/d1fc13996d841f6fb221eb90b0da9b9e880fed05/backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java#L130
not sure how much it matters but technically we are trying to emulate the RS message shape, and they have 3 possible scopes I believe: "item", "report" and "parameter"